pkg/kubelet/server/stats/summary.go

```diff
@@ -283,12 +284,27 @@ func (sb *summaryBuilder) containerInfoV2ToStats(
 			PageFaults:      &pageFaults,
 			MajorPageFaults: &majorPageFaults,
 		}
+		// availableBytes = memory  limit (if known) - workingset
+		if !isMemoryUnlimited(info.Spec.Memory.Limit) {
+			availableBytes := info.Spec.Memory.Limit - cstat.Memory.WorkingSet
```

Comment:
We do not support swap as of now. If/when we do, memory limit would not include swap, we would have to use the total limit - swap + ram